<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgWebAuth</class>
 <widget class="QDialog" name="CDlgWebAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authenticate</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>471</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Retry</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_headingLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heading</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>471</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_mainVerticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QGroupBox" name="m_pinGroupBox">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="m_pinLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_pinLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="m_confirmPinLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm PIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="m_confirmPinLineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="m_pinEntryErrorLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>441</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
